src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -385,7 +385,7 @@ UserInfoPopup::UserInfoPopup(bool closeAutomatically, QWidget *parent)
 
         QObject::connect(usercard.getElement(), &Button::leftClicked, [this] {
             QDesktopServices::openUrl("https://www.twitch.tv/popout/" +
```

Comment:
call this instead https://github.com/Chatterino/chatterino2/blob/fb9c3ad42ba4a7361d9c8889a1230a2baf71b1d8/src/util/Twitch.cpp#L8

Reply:
Thanks. I'll do a more thorough check before I push that.

Reply:
No longer required, reword channel passing